server/historian/packages/historian-base/package.json

```diff
@@ -22,12 +22,12 @@
     "tslint": "tslint 'src/**/*.ts'"
   },
   "dependencies": {
-    "@fluidframework/common-utils": "^0.24.0",
-    "@fluidframework/gitresources": "^0.1016.1",
-    "@fluidframework/protocol-definitions": "^0.1016.1",
-    "@fluidframework/server-services": "^0.1016.1",
+    "@fluidframework/common-utils": "^0.26.0",
+    "@fluidframework/gitresources": "^0.1017.1",
+    "@fluidframework/protocol-definitions": "^0.1017.1",
+    "@fluidframework/server-services": "^0.1017.1",
     "@fluidframework/server-services-core": "0.1014.0-7764",
```

Comment:
Just to confirm, this cannot be upgraded due to some interfaces moving from core to client, correct?

Reply:
Correct. The later versions don't have the interfaces which would cause build failures.